make_rc.py

```diff
@@ -0,0 +1,196 @@
+        prompt = " [Y/n] "
+    elif default == "no":
+        prompt = " [y/N] "
+    else:
+        raise ValueError("invalid default answer: '%s'" % default)
+
+    while True:
+        sys.stdout.write(question + prompt)
+        choice = input().lower()
+        if default is not None and choice == '':
+            return valid[default]
+        elif choice in valid:
+            return valid[choice]
+        else:
+            sys.stdout.write("Please respond with 'yes' or 'no' (or 'y' or 'n').\n")
+
+def work_on_release(next_release):
+    github_token = os.environ.get("GITHUB_TOKEN")
+    if not github_token:
+        print(github_token)
+        sys.stderr.write("Please, provide a read-only token to access Github using environment variable 'GITHUB_TOKEN'")
+
+    # Find matching milestone
+    g = Github(github_token)
+    repo = g.get_repo('conan-io/conan-vs-extension')
+    open_milestones = repo.get_milestones(state='open')
+    for milestone in open_milestones:
+        if str(milestone.title) == next_release:
+            # Gather pull requests
+            prs = [it for it in repo.get_pulls(state="all") if it.milestone == milestone]
+            sys.stdout.write("Found {} pull request for this milestone:\n".format(len(prs)))
+            for p in prs:
+                status = "[!]" if not p.is_merged() else ""
+                sys.stdout.write("\t {}\t{}\n".format(status, p.title))
+            
+            # Gather issues
+            issues = [it for it in repo.get_issues(milestone=milestone, state="all")]
+            sys.stdout.write("Found {} issues for this milestone:\n".format(len(issues)))
+            for issue in issues:
+                status = "[!]" if issue.state != "closed" else ""
+                sys.stdout.write("\t {}\t{}\n".format(status, issue.title))
+            
+            # Any open PR or issue?
+            if any([not p.is_merged() for p in prs]) or any([issue.state != "closed" for issue in issues]):
+                sys.stderr.write("Close all PRs and issues belonging to the milestone before making the release")
+                return
+            
+            # Modify the working directory
+            set_current_version(next_release)
+            write_changelog(next_release, prs)
+
+            # Commit current and checkout rc branch
+            # TODO: May work directly with the API
+            os.system("git add CHANGELOG.md")
+            os.system("git add Conan.VisualStudio/source.extension.cs")
+            os.system("git add Conan.VisualStudio/source.extension.vsixmanifest")
+
+            if query_yes_no("Commit change to 'dev' branch"):
+                os.system('git commit -m "close milestone {}"'.format(next_release))
+                os.system('git push')
+
+                os.system('git checkout -b rc-{}'.format(next_release))
+                os.system('git push')
+
+                sys.stdout.write("Now commit this branch and make the PR to master")
```

Comment:
it's already committed and pushed, what do you mean?

Reply:
All of this should happen in the repo, not in the local working copy